src/chttpd/test/eunit/chttpd_stats_tests.erl

```diff
@@ -0,0 +1,48 @@
+-module(chttpd_stats_tests).
+
+-include_lib("couch/include/couch_eunit.hrl").
+-include_lib("couch/include/couch_db.hrl").
+
+
+setup() ->
+    ok = meck:new(chttpd_stats, [passthrough]).
+
+teardown(_) ->
+    meck:unload(),
+    ok.
+
+
+chttpd_stats_test_() ->
+    {
+        "chttpd_stats tests",
+        {
+            setup,
+            fun chttpd_test_util:start_couch/0,
```

Comment:
Do we technically need to start / stop couch? Is it just to make config work? Then we can mock the config and make it return the default object

Reply:
no, but we also need couch_log. so started both for now